components/automate-ui/src/app/entities/automate-settings/automate-settings.model.ts

```diff
@@ -1,73 +1,141 @@
 export class JobSchedulerStatus {
-  running: boolean;
   jobs: IngestJob[];
 
-  constructor(running: boolean, ingestJobs: IngestJob[]) {
-    this.running = running;
+  constructor(ingestJobs: IngestJob[]) {
     this.jobs = ingestJobs;
   }
-
-  getJob(name: string): IngestJob {
-    return this.jobs.find((job: IngestJob) => job.name === name);
-  }
 }
 
 export interface ConfigureSettingsRequest {
   jobs: IngestJob[];
 }
 
 export interface RespJob {
-  running: boolean;
   name: string;
-  every: string;
+  nested_name?: string;
+  disabled: boolean;
+  recurrence: string;
   threshold: string;
-  last_run: Date;
-  next_run: Date;
-  last_elapsed: Date;
-  started_on: Date;
+  purge_policies: any; // needs specific type
+  last_elapsed?: Date;
+  next_due_at?: Date;
+  last_enqueued_at?: Date;
+  last_started_at?: Date;
+  last_ended_at?: Date;
 }
 
 export interface RespJobSchedulerStatus {
-  running: boolean;
-  jobs: RespJob[];
+  infra: {
+    jobs: RespJob[];
+  };
+  compliance: {
+    jobs: RespJob[];
+  };
+  event_feed: {
+    jobs: RespJob[];
+  };
+  services: {
+    jobs: RespJob[];
+  };
 }
 
 // IngestJobs is an enum that defines the list of jobs that the
 // JobScheduler has inside the ingest-service
 export enum IngestJobs {
-  // MissingNodes: Checks when a node hasn't check-in
-  // for a period of time
-  MissingNodes = 'missing_nodes',
+  //  EventFeed
+  EventFeedRemoveData = 'eventFeedRemoveData',
+  EventFeedServerActions = 'eventFeedServerActions',
 
-  // MissingNodesForDeletion: Checks when a node has been missing
-  // for a period of time
-  MissingNodesForDeletion = 'missing_nodes_for_deletion',
+  // Service Groups
+  ServiceGroupNoHealthChecks = 'serviceGroupNoHealthChecks',
+  ServiceGroupRemoveServices = 'serviceGroupRemoveServices',
+
+  // Client Runs
+  ClientRunsRemoveData = 'clientRunsRemoveData',
+  ClientRunsLabelMissing = 'clientRunsLabelMissing',
+  ClientRunsRemoveNodes = 'clientRunsRemoveNodes',
 
-  // DeleteNodes: Removes completely from elasticsearch nodes that
-  // have been marked for deletion
-  DeleteNodes = 'delete_nodes'
+  // Compliance
+  ComplianceRemoveReports = 'complianceRemoveReports',
+  ComplianceRemoveScans = 'complianceRemoveScans'
 }
 
 export class IngestJob {
-  running: boolean;
+  category: string;
```

Comment:
Here is a case where strong types are vital. To see why, before you make this change open this up in VSCode: https://github.com/chef/automate/blob/c2552860717826455b86591c8d821f33ac67b0f8/components/automate-ui/src/app/entities/automate-settings/automate-settings.requests.ts#L75-L98. If you hover over `job_settings` you will see it has type `any` even though `JobRequestBody` itself is strongly typed.

This needs to be either…
```suggestion
  category: 'infra' | 'compliance' | 'event_feed' | 'services';
```
or preferred, introduce an enum, e.g.:
```suggestion
  category: JobCategories;
```
(Once you make this change it will reveal an issue in that same block of code in the requests file.)